config/config.go

```diff
@@ -235,11 +235,12 @@ func (cfg *Configuration) GetCachedAssetURL(uuid string) string {
 }
 
 // Set the default config values for the viper object we are using.
-func SetupViper(v *viper.Viper) {
-	v.SetConfigName("pbs")
-	v.AddConfigPath(".")
-	v.AddConfigPath("/etc/config")
-
+func SetupViper(v *viper.Viper, filename string) {
```

Comment:
Little puzzled... what happened here?

Does this go away if you merge master into the branch? I'd've sworn we already merged these changes elsewhere.

Reply:
Ok, I am really puzzled as config/config.go was not part of the commit when I created it.